package.json

```
@@ -52,6 +52,10 @@
     }
   },
   "lint-staged": {
+    "*.{js,ts,tsx}": [
+      "eslint --ext .js,.tsx,.ts --cache --fix",
+      "prettier --write"
+    ],
     "*.{ts,tsx,json,scss}": [
       "prettier --write"
```

Question: this one could be replaced by the following, right? Since we already are running prettier via eslint.

```suggestion
    "*.{json,scss}": [
      "prettier --write"
```

---

You are right! Also removing the prettier --write in the rule above because eslint will fix those errors.